### ISSUE TITLE
A blank space is displayed instead of the "Screenshots" toolbar button

### ISSUE THREAD

**Issue by SoftVision-EmilPasca** [mozilla-services/screenshots#3125]
[Notes]:
- After installing the Screenshots addon provided in the email, the attached error message is displayed in the browser console:
`Screenshots error checking for Page Shot migration: Error: Could not establish connection. Receiving end does not exist.`
- After clicking on the blank space where the "Screenshots" toolbar button should be, the icon is displayed.
- On the unbranded Firefox release, when clicking on the blank space nothing happens and an error is displayed in the browser console:
`can't access lexical declaration `loadedPromise' before initialization`

[Affected versions]:
- Firefox 55.0b8 (Build ID 20170710085521)
- Firefox 56.0a1 (Build ID 20170714100307)

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Prerequisites]:
- Have a Firefox profile with the Screenshots addon provided in the email (10.4.4171) installed.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Observe the "Screenshots" toolbar button.

[Expected result]:
- The "Screenshots" toolbar button is correctly displayed.

[Actual result]:
- A blank space is displayed in place of the "Screenshots" toolbar button.

[Additional Notes]:
- Attached a screen recording of the issue:
![screenshots_icon_issue](https://user-images.githubusercontent.com/20083658/28214345-8f259e08-68b2-11e7-8615-d73b42dbedc1.gif)

**Comment by ianb** [comment]
I think I might have built the add-on incorrectly (without bootstrap.js), leading to the error in the console.  That wouldn't explain the icon problem though.

I'm worried the icon fix for 10.4.0 isn't working. If it went wrong it was supposed to lead to a black icon, which was undesirable but still usable, but obviously this is worse.

**Comment by 6a68** [comment]
I just installed the addon from the email into a fresh copy of today's Nightly (56.0a1 (2017-07-14) (64-bit)) on OSX. 

I don't see a bootstrap.js in the debugger when I load this addon, and I do see the communication error on startup.

I do, however, see the screenshots icon, and it works fine.

**Comment by 6a68** [comment]
I also see the screenshots icon in current Beta / OSX.

**Comment by SoftVision-PaulOiegas** [comment]
On Nightly is indeed working, but we are talking about Beta. Also, on Nightly the icon remains black after install and stays like that until you restart your browser. That's the behavior I have observed yesterday on #3100.

**Comment by 6a68** [comment]
Hmm. On current beta (55.0b9 (64-bit)) / OSX, I'm not able to repro the missing icon with either of the .xpis that Ian sent out last night

**Comment by 6a68** [comment]
My steps:

1. Open Firefox in a new profile
2. Go to about:config, set `extensions.signatures.required` to `false`, and double-check that `extensions.screenshots.system-disabled` is `false`
3. Go to about:addons, install the XPI file locally

Result: screenshots icon appears in toolbar, works fine. It is greyed out after I click the button once (when the overlay is shown).

**Comment by SoftVision-PaulOiegas** [comment]
Yes, it seems to be only Windows. Maybe Emil forgot to delete the other platforms when he logged the bug. We usually copy the bug format and add the new info.

**Comment by 6a68** [comment]
Ah, great. I'll grab my windows machine

**Comment by SoftVision-PaulOiegas** [comment]
@6a68 Nope, I'll retire what I said about Windows. Emil did not set `extensions.screenshots.system-disabled` is `false` when he tested. We were installing the provided xpi before enabling screenshots in Beta.

**Comment by 6a68** [comment]
😅

**Comment by johngruen** [comment]
I just reproed this on nightly with a new windows VM and a fresh firefox install...i wonder if the icon-starred-32-v2.svg is somehow invalid

**Comment by johngruen** [comment]
@SoftVision-PaulOiegas i figured out this issue, it appears the root el SVG el needs an explicit width and height declaration, _and_ `startBackground.js` needs a reference to the starred svg as well

**Comment by SoftVision-CosminMuntean** [comment]
I have retested this issue using a new custom Screenshots dev build from GitHub repository and is no longer reproducible on Beta 55 and latest Nightly (56.0a1) build.

I have also tested with the [Nightly try build](https://treeherder.mozilla.org/#/jobs?repo=try&revision=9c33620f97ab077ef202ff1af5ffb464dd1fc267) with Screenshots 10.7.0, created by Jared and the issue is not reproducible. 
But, I have tested this only on Windows, because I haven't found any Nightly build for Mac Os.
